src/middleware-set-current-time.js

```diff
@@ -0,0 +1,23 @@
+import videojs from 'video.js';
+
+// since VHS handles HLS and DASH (and in the future, more types), use * to capture all
+videojs.use('*', (player) => {
+  return {
+    setSource: (srcObj, next) => {
+      // pass null as the first argument to indicate that the source is not rejected
+      next(null, srcObj);
```

Comment:
I believe `setSource` and `setCurrentTime` will be called for all sources so this would error if we're not loading the vhs source handler (I think).

It will probably break in some way if `currentTime()` when loading a non-HLS source after HLS. The `player.vhs` property will still exist for that new source since we don't delete that on dispose.

Reply:
I thought middleware `setSource` was only to determine whether we run a middleware for a specific source?

Good catch on the checks for `player.vhs` property 🤔

Reply:
also, if we don't mind upping minimum version, `setSource` is now optional.
We should probably remove `player.vhs` when the source handler is disposed.